3/recipes search using ingredient

HamishAitken opened the pull request:
Working with multiple ingredients.

Marlon288 reviewed (commented):
Im confused, did you copy the index html and created a new one in your branch? Did you do it because it wasnt merged in the main branch? And if so what kind of changes did you do here apart from the search button?

HamishAitken commented:
> Im confused, did you copy the index html and created a new one in your branch? Did you do it because it wasnt merged in the main branch? And if so what kind of changes did you do here apart from the search button?

Yeah, copied the front end code to test. Added code to the index.js file, added buttons and altered some of the other html to work with our search implementation

HamishAitken commented:
After merging, ingredient search has stopped working.

Marlon288 commented:
We are trying to resolve the issue, its a Github problem. It has troubles recognizing the changes done. Thats why we changed the Base Branch to make github recalibrate and recalculate it. But it did not work, so we might just merge it to main.

Marlon288 commented on resources/css/details.css:
```diff
@@ -1,19 +1,19 @@
+  align-items: center;
+  height: 50px;
+}
 
```
No changes done here, so I will discards it

Marlon288 commented on resources/intro/readme.txt:
```diff
@@ -0,0 +1,40 @@
+﻿Congratulations, you have successfully downloaded font file! 
```
This file needs to be deleted, it is just a readme file and unneccasary

Marlon288 reviewed (approved):
This should be all fine for now
